Extensions/PrimitiveDrawing/shapepainterruntimeobject-pixi-renderer.js

```diff
@@ -47,9 +47,16 @@ gdjs.ShapePainterRuntimeObjectPixiRenderer.prototype.drawLine = function(x1, y1,
     this._graphics.endFill();
 };
 
+gdjs.ShapePainterRuntimeObjectPixiRenderer.prototype.drawLineV2 = function(x1, y1, x2, y2, thickness) {
+    this._graphics.lineStyle(thickness);
+    this._graphics.moveTo(x1, y1);
+    this._graphics.lineTo(x2,y2);
+    this._graphics.endFill();  
+};
+
 gdjs.ShapePainterRuntimeObjectPixiRenderer.prototype.drawEllipse = function(x1, y1, width, height) {
     this._graphics.beginFill(this._object._fillColor, this._object._fillOpacity / 255);
-    this._graphics.drawEllipse(x1, y1, width, height);
+    this._graphics.drawEllipse(x1, y1, width/2, height/2);
```

Comment:
It would be interesting to move the division by 2 in the renderer. Because otherwise it's a bit surprising to have the renderer and the object both having a "drawEllipse" method, but that accept different parameters.